cdf-core/cdf/js-modules/components/ButtonComponent.js

```diff
@@ -52,6 +83,7 @@ define(['amd!../lib/underscore', '../lib/jquery', './ActionComponent'],
      */
     enable: function() {
       this.placeholder('button').removeAttr('disabled');
+      this.placeholder('button').css({ opacity: 1 });
     },
```

Comment:
Probably best if we assign a css class to the button and have this style defined in cdf.css (not sure about the name). That way it can be overriden by implementors